ext/reflection/php_reflection.c

```diff
@@ -2996,7 +2996,7 @@ ZEND_METHOD(reflection_type, isBuiltin)
 	}
 	GET_REFLECTION_OBJECT_PTR(param);
 
-	RETVAL_BOOL(param->arg_info->type_hint != IS_OBJECT);
```

Comment:
@dragoonis it is used in `GET_REFLECTION_OBJECT_PTR` macro:

    #define GET_REFLECTION_OBJECT_PTR(target)                                                                   \
	GET_REFLECTION_OBJECT()  \
	target = intern->ptr;